### ISSUE TITLE
KEYPOD: Why does the appname need to be passed to the home screen

### ISSUE THREAD

**Issue by gjwgit** [anusii/keypod#60]
## Description

In lib/screens/home.dart we see the following doc and requirements to have `appname` passed to the `Home` widget. Why is that required as a parameter? Could we not determine the appname from the pubspec as in:

```
              final appNameVersion = await getAppNameVersion();
```

## So That

The template app is simple and demonstrative of the best approach.

## Closing Criteria

- [x] Decide if to remove the `appName` parameter

If removed:

- [ ] The appname is obtained within the Home class
- [ ] The appname is displayed in the AppBar title
- [ ] The appname is displayed in the About popup
- [ ] The app continues to function as before

## Detail

From `home.page`

```
/// Widget represents the home screen of the application.
///
/// It only requires [appName] to be passed to it during initialization.
/// This is because this page is designed to be work in offline as well.

class Home extends StatefulWidget {
  /// Initialise widget variables

  const Home({required this.appName, super.key});
  final String appName;
```

**Comment by gjwgit** [comment]
Make it optional as a demo of obtaining name from pubspec but can be overridden

**Comment by cdawei** [comment]
After a discussion in the standup meeting, we decide to do the following:
1. Get the app name directly from `pubspec.yml` (using an existing function)
2. Replacing blank spaces with underscores and lowercasing all characters in the app name
3. Remove the `appName` argument from the `Home` widget.

**Comment by cdawei** [comment]
Partially fixed in pull request #67 and more work needed.

**Comment by cdawei** [comment]
Fixed the remaining issue in pull request https://github.com/anusii/keypod/pull/74
